packages/components/RadioGroup/src/RadioButton.settings.ts

```diff
@@ -61,23 +62,22 @@ export const settings: IComposeSettings<IRadioButtonType> = [
       style: {
         marginTop: 3,
         borderStyle: 'dashed',
-        borderColor: 'transparent',
         borderWidth: 1
       }
     },
-    // This still uses the innerCircle's 'hover' style instead of the 'focused' style when need be. I'm commenting out'
-    _precedence: ['disabled', 'hovered', 'selected'],
+    // This still uses the innerCircle's 'hover' style instead of the 'focused' style when need be. I'm commenting out
+    _precedence: ['disabled', 'hovered', 'isFocused', 'selected'],
```

Comment:
Rename this to 'focused' to keep the naming convention the same across our components?

Reply:
Changing this now. I initially thought there might be some negative interaction between pressable's 'focused' and this, but it looks to be fine.